functional_tests/scylla_operator/test_functional.py

```diff
@@ -332,3 +332,21 @@ def test_scylla_operator_pods(db_cluster: ScyllaPodCluster):
 
     not_running_pods = ','.join([pods_info[0] for pods_info in scylla_operator_pods if pods_info[1] != 'Running'])
     assert not not_running_pods, f'There are pods in state other than running: {not_running_pods}'
+
+
+def test_startup_probe_in_scylla_pods(db_cluster: ScyllaPodCluster):
+    pods = db_cluster.k8s_cluster.kubectl(f'get pods -A -l "scylla/cluster=sct-cluster" -o yaml')
+
+    no_startup_prob_pods = []
+    for pod in yaml.load(pods.stdout)["items"]:
+        if "spec" not in pod:
```

Comment:
Let's squash these statements to one:
```suggestion
        if "spec" not in pod or "containers" not in pod["spec"]:
            continue
```

Reply:
done